### DISCUSSION TITLE
"stars=" should consider significant digits

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20757] [Ideas]
currently the "stars=" (or "star=") search query only returns maps of the exact star rating (ie "stars=4" only returns 4.00* maps)
This should be changed to consider significant digits (ie "stars=4" would return all beatmaps from 4.00* to 4.99*, and "stars=4.5" would return all beatmaps from 4.50* to 4.59*)
would be nice in the browser instead of always having to type the range (ie "stars>4 stars<5")

**Comment by ghost** [comment]
i think the format "stars=4-5" to get beatmaps of 4.00*-5.00* would make more sense, because with your case, "stars=4", "stars=4.0" and "stars=4.00" would all yield different results, which doesnt make much sense to me

**Reply by Doksha** [reply]
4.00 would be within the search of 4, 4 is simply a broader search

**Reply by frenzibyte** [reply]
It doesn't make sense mathematically, I'm in favour of explicitly specifying ranges rather than redefining the meaning of `4` and `4.00`.
